package.json

```diff
@@ -29,6 +30,10 @@
   "author": "",
   "license": "MIT",
   "devDependencies": {
+    "//": [
```

Comment:
🛑 Hmm - this got moved. Maybe you could move it out of `devDependencies` and update to specifically reference `enzyme-adapter-react-16`?

Reply:
Thanks for noticing this.  I guess I'll avoid running `yarn add` in the future.

Reply:
I guess if we follow the same pattern as the package.json in webapp this shouldn't be an issue.